test_project/test_project/settings.py

```diff
@@ -142,6 +145,20 @@
 
 GRAPPELLI_ADMIN_TITLE = "Dancerfly"
 
+STATICFILES_FINDERS = (
+    'django.contrib.staticfiles.finders.FileSystemFinder',
+    'django.contrib.staticfiles.finders.AppDirectoriesFinder',
+    # other finders..
+    'compressor.finders.CompressorFinder',
+)
+
+_BOOTSTRAP_SASS_SUBPATH = "/gems/bootstrap-sass-3.3.4.1/assets/stylesheets/"
+STATICFILES_DIRS = [x + _BOOTSTRAP_SASS_SUBPATH for x in os.environ['GEM_PATH'].split(":") if os.path.isdir(x + _BOOTSTRAP_SASS_SUBPATH)]
+
+COMPRESS_PRECOMPILERS = (
+    ('text/sass', 'django_libsass.SassCompiler'),
```

Comment:
Why do we need this? The example in the docs has `('text/x-sass', 'sass {infile} {outfile}')`. (I'm wondering about sass compiler settings, because it would be nice to have it compile uncompressed in development.)